src/NumericalAlgorithms/DiscontinuousGalerkin/Actions/ApplyFluxes.hpp

```diff
@@ -59,27 +74,23 @@ struct ApplyBoundaryFluxesGlobalTimeStepping {
                     const ArrayIndex& /*array_index*/,
                     const ActionList /*meta*/,
                     const ParallelComponent* const /*meta*/) noexcept {
-    static_assert(not Metavariables::local_time_stepping,
-                  "ApplyBoundaryFluxesGlobalTimeStepping cannot be used with "
-                  "local time-stepping.");
-
     using system = typename Metavariables::system;
     constexpr size_t volume_dim = system::volume_dim;
     using variables_tag = typename system::variables_tag;
-    using dt_variables_tag = db::add_tag_prefix<Tags::dt, variables_tag>;
+    using dt_variables_tag =
```

Comment:
Do you want to rename this?  (Up to you.)

Reply:
I think it's a fine name for this local variable